tika-parsers/src/main/java/org/apache/tika/parser/recognition/ObjectRecognitionParser.java

```diff
@@ -87,7 +90,13 @@ public int compare(RecognisedObject o1, RecognisedObject o2) {
     @Field
     private int topN = 2;
 
-    private ObjectRecogniser recogniser = new TensorflowRESTRecogniser();
+    @Field
+    private int captions;
+
+    @Field
+    private int maxCaptionLength;
```

Comment:
No need to declare `captions` and `maxCaptionLength` here right?
If the only purpose here is to log them, why not log them inside the implementation of `ObjectRecogniser` itself ?